news/2019-02-05-new-changes-to-star-rating-performance-points.md

```diff
@@ -0,0 +1,82 @@
+
+## Streams and High BPM
+
+Besides the change to wide angle jumps, there has been an adjustment to how streams are handled by the algorithm. Many players who are capable of incredible speeds find their ranks are unfairly low given their unique skillset. To promote these unique skillsets, streams exceeding 200 BPM now have an increased reward that more accurately represents their difficulty.
+
+Able to FC 300 BPM stacks with your laptop keyboard? This is the change you've been waiting for. 
+
+A small nerf has also been applied to stream patterns with very high spacing between each hitcircle. These are those streams found in maps such as [HONESTY](https://osu.ppy.sh/beatmapsets/586121#osu/1241370) and [Sidetracked Day](https://osu.ppy.sh/beatmapsets/728276#osu/1537566). While the nerfs are noticable, these plays are still quite impressive and worth a lot, just not quite as much as they were worth before.
+
+## Slider Buffs
+
+Are you a fan of playing those impressive and previously underweighted tech maps with high slider velocity? 
+
+Well, say no more! 
+
+While the specifics are quite complicated, in general, these fast sliders are now more accurately judged by the algorithm. Not only that, but maps with long sliders that have jumps between them also received a large buff compared to before.
+
+This is a change that will help buff many maps with high slider velocity. While this change will not be noticeable on many maps, the ones that do benefit may see substantial buffs, such as [this one](https://osu.ppy.sh/beatmapsets/753365#osu/1586083).
+
+## Flashlight Rework
+
+One of the exciting introductions that will be coming with this patch is the change to flashlight. Flashlight has been a seldom chosen mod for competitive play outside of very short beatmaps since PPv2's inception back in 2014 (more information can be found [here](https://osu.ppy.sh/help/wiki/Performance_Points)). 
+
+<todo: add user-friendly description of this change>
+
+## Hidden Rework
+
+Yes, HD is changing once again. With the introduction of the high BPM buff, we wanted to make changes to HD so that maps with fast stacked streams weren't overly buffed when HD is applied. 
+
+Instead of the old system, which primarily buffed speed (as speed was primarily spaced streams), HD now rewards accuracy more, with both aim and speed PP scaling off of the approach rate of the map. The lower the AR the more reward.
+
+Are you good at playing with EZ and HD? You'll find a notable increase in performance points for these plays after the update. 
+
+## Misc PP Changes
+
+These are the more minor technical changes that we introduced to fix previous issues and keep things balanced.
+
+- Sliders had excess travel distance due to an off-by-one error during positional calculations and due to the slider end always being considered as a tick.
+- The length of sliders was only considered if the next hitobject was also a slider.
```

Comment:
In a non-technical context, "hitobject" should probably be separated into two words.